apps/records-pages/src/api/TransactionalLimit/route.ts

```diff
@@ -0,0 +1,22 @@
+import type { TransactionalLimitInterface } from 'src/api/TransactionalLimit/interfaces';
+import {
+  GetTransactionalLimitOperatorStateTable,
+  GetTransactionalLimitTable
+} from 'src/api/TransactionalLimit/service';
+
+export async function GET(): Promise<TransactionalLimitInterface> {
+  const [operatorStateRes, transactionalLimitsRes] = await Promise.all([
+    GetTransactionalLimitOperatorStateTable(),
+    GetTransactionalLimitTable()
+  ]).catch((error) => {
+    throw new Error(error);
+  });
```

Comment:
Este throw podemos remover, não fez o efeito desejado 

```suggestion
  ]);
```